<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1074</width>
    <height>646</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="myGLWidget" name="openGLWidget"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1074</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="open"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>编辑</string>
    </property>
    <addaction name="point"/>
    <addaction name="line"/>
    <addaction name="face"/>
    <addaction name="loadTex"/>
    <addaction name="showTex"/>
    <addaction name="setLight"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>材质设置</string>
    </property>
    <addaction name="action_3"/>
    <addaction name="action_4"/>
    <addaction name="action_5"/>
    <addaction name="action_6"/>
    <addaction name="action_7"/>
    <addaction name="action_8"/>
    <addaction name="action_9"/>
    <addaction name="action_10"/>
    <addaction name="action_11"/>
    <addaction name="action_12"/>
    <addaction name="action_13"/>
    <addaction name="action_14"/>
    <addaction name="action_15"/>
    <addaction name="action_16"/>
    <addaction name="action_17"/>
    <addaction name="action_18"/>
    <addaction name="action_19"/>
    <addaction name="action_20"/>
    <addaction name="action_21"/>
    <addaction name="action_22"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_4"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="open">
   <property name="text">
    <string>打开</string>
   </property>
  </action>
  <action name="point">
   <property name="text">
    <string>点模型</string>
   </property>
  </action>
  <action name="line">
   <property name="text">
    <string>线模型</string>
   </property>
  </action>
  <action name="face">
   <property name="text">
    <string>面模型</string>
   </property>
  </action>
  <action name="loadTex">
   <property name="text">
    <string>贴图设置</string>
   </property>
  </action>
  <action name="showTex">
   <property name="text">
    <string>显示贴图</string>
   </property>
  </action>
  <action name="action_4">
   <property name="text">
    <string>青铜</string>
   </property>
  </action>
  <action name="action_5">
   <property name="text">
    <string>亮青铜</string>
   </property>
  </action>
  <action name="action_6">
   <property name="text">
    <string>铬</string>
   </property>
  </action>
  <action name="action_7">
   <property name="text">
    <string>铜</string>
   </property>
  </action>
  <action name="action_8">
   <property name="text">
    <string>亮铜</string>
   </property>
  </action>
  <action name="action_9">
   <property name="text">
    <string>金</string>
   </property>
  </action>
  <action name="action_10">
   <property name="text">
    <string>亮金</string>
   </property>
  </action>
  <action name="action_11">
   <property name="text">
    <string>白蜡</string>
   </property>
  </action>
  <action name="action_12">
   <property name="text">
    <string>银</string>
   </property>
  </action>
  <action name="action_13">
   <property name="text">
    <string>亮银色</string>
   </property>
  </action>
  <action name="action_14">
   <property name="text">
    <string>翡翠</string>
   </property>
  </action>
  <action name="action_15">
   <property name="text">
    <string>碧玉</string>
   </property>
  </action>
  <action name="action_16">
   <property name="text">
    <string>黑曜石</string>
   </property>
  </action>
  <action name="action_17">
   <property name="text">
    <string>珍珠</string>
   </property>
  </action>
  <action name="action_18">
   <property name="text">
    <string>红宝石</string>
   </property>
  </action>
  <action name="action_19">
   <property name="text">
    <string>绿宝石</string>
   </property>
  </action>
  <action name="action_20">
   <property name="text">
    <string>黑塑料</string>
   </property>
  </action>
  <action name="action_21">
   <property name="text">
    <string>黑橡胶</string>
   </property>
  </action>
  <action name="action_22">
   <property name="text">
    <string>紫罗兰</string>
   </property>
  </action>
  <action name="setLight">
   <property name="text">
    <string>打开灯光</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>黄铜</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>myGLWidget</class>
   <extends>QOpenGLWidget</extends>
   <header>myglwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
